pallets/session-benchmarking/src/lib.rs

```diff
@@ -39,6 +39,4 @@ benchmarks! {
 		let proof: Vec<u8> = vec![0,1,2,3];
 		let _t = pallet_session::Pallet::<T>::set_keys(RawOrigin::Signed(caller.clone()).into(), keys, proof);
 	}: _(RawOrigin::Signed(caller))
-
 }
-impl_benchmark_test_suite!(Pallet, crate::mock::new_test_ext(), crate::mock::Test, extra = false,);
```

Comment:
This could never work and was not compiling in the first place either, because the `mock` of `pallet_session` cannot be used here.